libr/cons/dietline.c

```diff
@@ -536,8 +536,9 @@ static void selection_widget_select() {
 }
 
 static void selection_widget_update() {
-	if (I.completion.argc == 0 ||
-		(I.completion.argc == 1 && I.buffer.length >= strlen (I.completion.argv[0]))) {
+	int argc = r_pvector_len (&I.completion.args);
+	const char **argv = I.completion.args.v.a;
```

Comment:
Isn’t there a method to get this? I don’t like accessing internal fields in this way

Reply:
Now there is